src/char/char.c

```diff
@@ -116,7 +116,7 @@ char char_reg_num_db[32] = "char_reg_num_db";
 char char_achievement_db[256] = "char_achievements";
 
 static struct char_interface char_s;
-struct char_interface *chr;
+struct char_interface *chr; // lgtm[cpp/short-global-name]
```

Comment:
I'm not a friend of explicitly suppressing warnings.
Actually I accidentally use names of global interface variables for local variables every now and then... :scream:
In my opinion we should at least consider renaming them to be more exclusive/descriptive.

Reply:
shadowing should be detected by compiler, even if visual studio cant do this, i think gcc on ravis will fail if you will use one of global variables. same for non global vars

Reply:
Sure, you're right. The Travis builds friendly remind me if I did it again. :grin:
But nonetheless I think renaming should be considered/discussed.